src/main/java/seedu/duke/exceptions/calendar/DuplicateTaskException.java

```diff
@@ -2,6 +2,10 @@
 
 import seedu.duke.exceptions.ClickException;
 
+/**
+ * Represents exception thrown when index of task
+ * to delete is not found.
+ */
```

Comment:
Javadoc(exception thrown when index of task to delete is not found) doesn't seem to match what the exception is supposed to do(in case duplicate task ?)

Reply:
changed this